### ISSUE TITLE
Understanding the `interval` column

### ISSUE THREAD

**Issue by damianooldoni** [riparias/vmm-macrophytes-occurrences#10]
**Question to VMM**.

The sheet `vegetatie-ontwikkeling` has a column called `interval` with a controlled vocabulary: `0-10m`, `10-20m`,`20-30m`, ... up to `90-100m`. These values cannot be related to depth as the depth of the waterbodies is never as deep as 100m (most of them are 1 to 2 meters deep). Is it the distance to the banks? Is it the distance upstream or downstream?

**Comment by damianooldoni** [comment]
Based on pdf in attachment, the data are related to 100m transects. They are then divided in 10 subtransects, each of them 10 meter long. The XY coordinates of the 100m transect is the most downstream point. Original text in Dutch from pdf:
> De xy-coördinaten worden ter plaatse zo precies mogelijk met de GPS bepaald op het meest
stroomafwaartse punt van het traject.

[GP_3111_v7.pdf](https://github.com/riparias/vmm-macrophytes-occurrences/files/8605558/GP_3111_v7.pdf)
.

**Comment by peterdesmet** [comment]
Corrected in 503059b3d65820f2d98248b6d67fba9770c20c89. Expressed as:

```
21327837,submerged plants 0-10m upstream,frequent to abundant,,
21327837,submerged plants 10-20m upstream,frequent to abundant,,
21327837,submerged plants 20-30m upstream,frequent to abundant,,
21327837,submerged plants 30-40m upstream,frequent to abundant,,
21327837,submerged plants 40-50m upstream,frequent to abundant,,
21327837,submerged plants 50-60m upstream,frequent to abundant,,
21327837,submerged plants 60-70m upstream,frequent to abundant,,
21327837,submerged plants 70-80m upstream,frequent to abundant,,
21327837,submerged plants 80-90m upstream,frequent to abundant,,
21327837,submerged plants 90-100m upstream,frequent to abundant,,
```

**Comment by peterdesmet** [comment]
Even though starting position is precise, the coordinate uncertainty radius is set to 100m (the maximum distance walked) because occurrences are collected along the way.
